std_msgs/Header header
geometry_msgs/Vector3 force
geometry_msgs/Vector3 moment
std_msgs/Float32 wing_left
std_msgs/Float32 wing_right